features/nanostack/mbed-mesh-api/mbed-mesh-api/WisunInterface.h

```diff
@@ -185,24 +185,31 @@ class WisunInterface : public MeshInterfaceNanostack {
     /**
      * \brief Set Wi-SUN network size.
      *
-     * Function stores new parameters to mbed-mesh-api and uses them when connect() is called next time.
-     * If device is already connected to the Wi-SUN network then device will restart network discovery after
-     * changing the network size.
+     * Function stores network size parameter to the mbed-mesh-api and uses it when connect() is called for the next
+     * time. If a device is already connected to the Wi-SUN network, then the device will restart network discovery
+     * after changing the network size.
+     *
+     * It is recommended to set the correct network size because some Wi-SUN network configuration parameters are
+     * adjusted based on the selected network size. A network configured for a small amount of devices may not work
+     * optimally for large number of devices. This is because the network bandwidth is divided with all the devices in
+     * the network. Enough bandwidth must be reserved for application data usage as well as the Wi-SUN network
+     * operations. In addition, the application should adapt to the network characteristics by using the InternetSocket
+     * methods get_stagger_estimate_to_address() and get_rtt_estimate_to_address().
+     *
+     * The network size is measured as hundreds of devices that are expected to join to the network. For example,
+     * for a 400-device network set network size to 4.
      *
-     * Default value: medium
-     * Small network size: less than hundred devices
-     * Medium network size: hundreds of devices
-     * Large network size: thousands of devices
-     * Certificate: used on testing
+     * The Wi-SUN stack will automatically adjust timing and RPL configuration values based on the selected network
+     * size and data rate. If a customized timing or RPL values are needed, the APIs below should be invoked after
+     * changing the network size:
+     * - set_timing_parameters() to set timing settings to the Wi-SUN interface.
+     * - rpl_parameters_set() to set RPL settings to the Border Router interface.
      *
-     * When network size is changed, it will override all or some of the following configuration values:
-     * - Timing settings set by set_timing_parameters() of the Wi-SUN interface.
-     * - RPL settings set by rpl_parameters_set() of the Border Router interface.
+     * By default the Wi-SUN stack is configured to use a few hundreds of devices.
```

Comment:
That's seems simple and clear, thanks for the update!
Just one question, if the "by default ... a few hundreds"  should be  "4 means 4 hundreds" as default?